spyder/plugins/findinfiles/widgets/main_widget.py

```diff
@@ -142,6 +143,13 @@ def __init__(self, name=None, plugin=None, parent=None):
             path_history = [path_history]
 
         # Widgets
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "find_empty",
+            _("You haven't searched for anything yet."),
```

Comment:
This sounds a bit odd to me as a non-native speaker. What about `Nothing has been searched for yet`?

Reply:
That sounds rather odd to me as a native speaker, hehe. I'm not sure else to say it, as while its not perfect, "Nothing searched for yet" is the only really idiomatic way I can think of to say this.

Not conclusive, but I can find [3 hits for the exact match "Nothing searched for yet"](https://www.google.com/search?q=%22Nothing+searched+for+yet%22), all application UI text similar to this, and [4 exact hits for your variation](https://www.google.com/search?q=%22Nothing+has+been+searched+for+yet%22), all four prose or comments rather than UI text.

Reply:
To me, "nothing searched for yet" and "nothing has been searched for yet" both sound fine, the latter is a full sentence and the former is the short form with verbs omitted, suitable for titles and similar. It's the same as "no plots generated yet" being short for "no plots have been generated yet". For consistency, I would go for the short form here as well.